loader/app/loaders/loader_cclw_v2/extract/test_main.py

```diff
@@ -4,7 +4,15 @@
 
 
 def test_extract():
-    data_dir = Path(__file__).parent.resolve() / ".." / ".." / ".." / ".." / "data"
+    csv_path = (
+        Path(__file__).parent.resolve()
+        / ".."
+        / ".."
+        / ".."
+        / ".."
+        / "data"
+        / "cclw_new_format_20220503.csv"
+    )
 
-    policies_fe = extract(data_dir)
-    assert len(policies_fe) == 87
+    policies_fe = extract(csv_path)
+    assert len(policies_fe) == 1371
```

Comment:
ooh, so many rows now...

Reply:
but the wrong number because i found another error in the input data 😂